apps/ssr-tests-v9/package.json

```diff
@@ -14,6 +14,7 @@
     "code-style": "just-scripts code-style",
     "lint": "just-scripts lint",
     "test": "jest --passWithNoTests",
+    "posttest": "node -r @fluentui/scripts/ts-node-register ./src/test.ts",
```

Comment:
This looks strange IMO 🤔 I can update it to:

```
    "test": "jest && node -r @fluentui/scripts/ts-node-register ./src/test.ts",
```

Reply:
I prefer inlining the second test command within the `test` script

Reply:
tbf you can even just create a jest test and do the same things as in `test.ts`

Reply:
Inlined the script.